src/log.js

```diff
@@ -57,7 +57,18 @@ function respectProgressBars(commitLogs) {
 }
 
 function getPrefix(chalkColor) {
-  return chalkColor('[') + chalk.gray('exp') + chalkColor(']');
+  const timestamp = new Date().toLocaleTimeString().padStart(11, '0');
```

Comment:
Padding with zero will look weird in many locales. For example, this is how it looks like on my system (using the `en-FI` locale):
<img width="454" alt="screen shot 2018-02-06 at 17 45 13" src="https://user-images.githubusercontent.com/497214/35868712-ff0e2d1a-0b65-11e8-919e-a4dce0faf69c.png">

Reply:
Ah yeah, good call - that doesn't look great. We'll definitely need a more locale-agnostic format.

Reply:
What about something like this?
```js
const maxLength = new Date('12-31-1970 23:59:59').toLocaleTimeString().length;
const timestamp = new Date().toLocaleTimeString().padStart(maxLength);
```
It pads with space (the default) instead of zero and should only ever need to pad one character since we're only padding to the length of a timestamp that should be of maximum width.

(Note: I'd probably pull the `maxLength` calculation out of the prefix function as there's no need to compute that for every log line.)